tcms/core/utils/__init__.py

```diff
@@ -60,7 +60,7 @@ def clean_request(request, keys=None):
     """
```

Comment:
> I guess you mean to remove sorting by sortkey, right?

No, I don't mean that. See the diff above. I mean to remove `clean_request()` and also remove the code which sorts by some other field but leave sorting via `sortkey`. The reason I propose this is because only sorting by `sortkey` is currently used. See diff from previous comment.